Healthy/Classes/Modules/Dashboard/Views/FoodTagsView/TagCollectionViewCell/TagCollectionViewCell.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+class TagCollectionViewCell: UICollectionViewCell {
+
+    // MARK: Outlets
+
+    @IBOutlet weak var backView: UIView! {
+        didSet {
+            backView.layer.cornerRadius = 10
+        }
+    }
```

Comment:
```suggestion
    @IBOutlet weak var backView: UIView!
```
Please move any view configurations updates outside didSet...